WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -840,8 +840,9 @@ private bool TryParseUserFee(out decimal userFee)
 				userFeeText is { }
 				&& !userFeeText.Contains(",")
 				&& decimal.TryParse(userFeeText, out userFee)
-				&& (userFee * 1_000) < Constants.MaxSatoshisSupply
-				&& userFee > 0;
+				&& Math.Ceiling(userFee) == Math.Floor(userFee)
+				&& Math.Round(userFee) < MaxSatoshisSupplyLimit
+				&& Math.Round(userFee) > 0;
```

Comment:
Decimal numbers like `0.5` or `1.5` are invalid: 
`1.9999999999999999999999999999`   -> invalid
`1.99999999999999999999999999999` -> valid

This should fix it:
```suggestion
				&& userFee < MaxSatoshisSupplyLimit
				&& userFee > 0;
```

Shouldn't we use some max length property with these controls?

Reply:
MaxLength is out of scope for this PR  i think